app/models/katello/installed_package.rb

```diff
@@ -2,5 +2,7 @@ module Katello
   class InstalledPackage < Katello::Model
     has_many :hosts, :through => :host_installed_packages, :class_name => "::Host"
     has_many :host_installed_packages, :class_name => "Katello::HostInstalledPackage", :dependent => :destroy, :inverse_of => :installed_package
+
+    scoped_search :in => :host_installed_packages, :on => :name
```

Comment:
It seems to work although I am not sure if the `:in => :host_installed_packages` is meaningless.